lib/listening.js

```diff
@@ -1,14 +1,40 @@
 // Native
+const { existsSync, statSync } = require('fs')
 const path = require('path')
-const debounce = require('debounce')
 
 // Packages
+const ignoredDirectories = require('ignore-by-default').directories()
 const { write: copy } = require('clipboardy')
-const ip = require('ip')
-const chalk = require('chalk')
-const boxen = require('boxen')
+const getIgnoredFromGit = require('ignored')
 const { watch } = require('chokidar')
+const anymatch = require('anymatch')
```

Comment:
using `anymatch` because i'ts a `chokidar` dependency

Reply:
Is `anymatch` listed in the package.json list of dependencies? If not, this feels brittle.

Reply:
yep https://github.com/zeit/micro-dev/pull/39/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R42